www/resources/views/pages/testhistoryadmin.blade.php

```diff
@@ -72,7 +72,7 @@
                         'labelTxt' => $labelTxt,
                     ])
                 <tr>
-                    <th>
+                    <th class="history_checkbox">
```

Comment:
why history_checkbox on a th?

Reply:
It's meant to align the content. Maybe a better name like 'select_test' or something?

Reply:
yes please let's find a better name